src/asymmetric_cipher.c

```diff
@@ -106,7 +107,9 @@ static int p11prov_rsaenc_encrypt_init(void *ctx, void *provkey,
     P11PROV_OBJECT *obj = (P11PROV_OBJECT *)provkey;
 
     p11prov_debug("encrypt init (ctx=%p, key=%p, params=%p)\n",
-                  ctx, provkey, params);
+                  ctx,
+                  provkey,
+                  params);
 
```

Comment:
But this one is a bit gross, is there a way to make indentation greedy and stay on the line until 80 chars are hit and only then start a new line?

In rare cases having one argument per line is better, but it is generally wasteful.